docs/gro-ontology.rst

```diff
@@ -35,7 +35,7 @@ The above information can be retrieved using the :meth:`api.client.gro_client.Gr
 
   client.lookup('items', 270)
 
-`See this Getting Started section for help on finding entities by name or by id. <understanding-data-responses#look-up>`
+See this Getting Started section for help on finding entities by name or by id. `Understanding Data Responses#lookup <understanding-data-responses#lookup>`_
```

Comment:
I think we want this to read more like:

`See this Getting Started section for help on finding entities by name or by id.<understanding-data-responses#lookup>`

<understanding-data-responses#lookup> is not text that needs to be seen by the user

Reply:
![image](https://user-images.githubusercontent.com/54555734/76435357-d0fe2f80-63ae-11ea-8374-3ce1fbf19b6a.png)
this is what is generated in html

Reply:
Right, but I don't think it should look like that. "Understanding Data Responses#lookup" doesn't look right. It should be more along the lines of what it used to look like:

![image](https://user-images.githubusercontent.com/45766323/76436285-6ccb7380-638e-11ea-8818-117ff044e028.png)

Reply:
gotcha fixed to updated link style:
![image](https://user-images.githubusercontent.com/54555734/76438703-410eb480-63b3-11ea-9f26-98c32329ae52.png)